src/api/web.ts

```diff
@@ -33,3 +33,14 @@ export function login(req: express.Request, res: express.Response): express.Resp
     return res.status(200).json({ auth: false });
   }
 }
+
+export function getUserOrganizations(
+  req: express.Request,
+  res: express.Response): express.Response {
+    let username = req.body.username;
+    let auth = getAuth();
+
+    return res.status(200).json({
+      organizations:
+        auth.organizations.filter(org => org.members.findIndex(u => u.name === username) !== -1)});
```

Comment:
maybe it would be nicer if you apply this to a variable before returning into response